### PR TITLE
drop support for Go 1.16 and 1.17

### PR DESCRIPTION
**Pull request by marten-seemann**
This is in draft status until Go 1.19 is released.

### CODE REVIEW

**Comment by marten-seemann** [comment]
Go 1.19 was released earlier today: https://go.dev/doc/go1.19. 🎉

@lucas-clemente, can you please review?

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In addition the RFCs listed above, it currently implements the [IETF QUIC draft-
 
 ## Guides
 
-*We currently support Go 1.16.x, Go 1.17.x, and Go 1.18.x.*
+*We currently support Go 1.18.x.*
```
Comment (marmeladema) [inline README.md:14]: Should this mention Go 1.19.x?

**Comment by marmeladema** [comment]
Support for Go 1.19 would be really helpful! Thank you for doing this.

File: README.md (hunk 1 above)
Comment (marten-seemann) [reply README.md]: Thank you! Fixed.

**Pull request merged**